test/units/testsuite-48.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -ex
+
+test_rule="/run/udev/rules.d/48-test.rules"
+
+setup() {
+    mkdir -p "${test_rule%/*}"
+    cp -f /etc/udev/udev.conf /etc/udev/udev.conf.bckp
+    echo 'KERNEL=="lo", SUBSYSTEM=="net", PROGRAM="/usr/bin/sleep 10"' > "${test_rule}"
+    echo "event_timeout=5" >> /etc/udev/udev.conf
```

Comment:
Unfortunately I think 5 seconds is still too short for Ubuntu CI.  I suggest using event_timeout=30 at least, and sleep 60.